lib/elixir_auth_google.ex

```diff
@@ -78,6 +78,7 @@ defmodule ElixirAuthGoogle do
   This is the URL you need to use for your "Login with Google" button.
   See step 5 of the instructions.
   """
+  @spec generate_oauth_url(binary()) :: String.t()
```

Comment:
I was wondering if it'd make more sense to keep both arg and output as `String.t()` 🤔

Reply:
Yeah, string for clarity. 👌